src/stp/new_skills/Kick.cpp

```diff
@@ -15,13 +15,15 @@ Status Kick::onUpdate(const StpInfo &info) noexcept {
     // Set kick command
     command.set_kicker(true);
     command.set_chip_kick_vel(kickVelocity);
+    command.set_chip_kick_forced(true);
 
     // Set angle command
     command.set_w(info.getRobot().value()->getAngle());
 
     publishRobotCommand();
 
-    if (info.getBall()->get()->getVelocity().length() > stp::control_constants::HAS_KICKED_ERROR_MARGIN) {
+    if (info.getBall()->get()->getVelocity().length() > stp::control_constants::HAS_KICKED_ERROR_MARGIN &&
+            info.getRobot().value()->getAngleDiffToBall() <= control_constants::GO_TO_POS_ANGLE_ERROR_MARGIN * M_PI) {
```

Comment:
This seems like a pretty unrelated change. How does Go TO POS affect the kick skill?